unity-renderer/Assets/DCLPlugins/ECS7/ECSComponents/Tween/ECSTweenHandler.cs

```diff
@@ -0,0 +1,216 @@
+                       && lastModel.Move.End.Equals(targetModel.Move.End);
+        }
+    }
+
+    private Ease SDKEasingFunctinToDOTweenEaseType(EasingFunction easingFunction)
+    {
+        switch (easingFunction)
+        {
+            case EasingFunction.TfEaseinsine:
+                return Ease.InSine;
+            case EasingFunction.TfEaseoutsine:
+                return Ease.OutSine;
+            case EasingFunction.TfEasesine:
+                return Ease.InOutSine;
+            case EasingFunction.TfEaseinquad:
+                return Ease.InQuad;
+            case EasingFunction.TfEaseoutquad:
+                return Ease.OutQuad;
+            case EasingFunction.TfEasequad:
+                return Ease.InOutQuad;
+            case EasingFunction.TfEaseinexpo:
+                return Ease.InExpo;
+            case EasingFunction.TfEaseoutexpo:
+                return Ease.OutExpo;
+            case EasingFunction.TfEaseexpo:
+                return Ease.InOutExpo;
+            case EasingFunction.TfEaseinelastic:
+                return Ease.InElastic;
+            case EasingFunction.TfEaseoutelastic:
+                return Ease.OutElastic;
+            case EasingFunction.TfEaseelastic:
+                return Ease.InOutElastic;
+            case EasingFunction.TfEaseinbounce:
+                return Ease.InBounce;
+            case EasingFunction.TfEaseoutbounce:
+                return Ease.OutBounce;
+            case EasingFunction.TfEasebounce:
+                return Ease.InOutBounce;
+            case EasingFunction.TfEaseincubic:
+                return Ease.InCubic;
+            case EasingFunction.TfEaseoutcubic:
+                return Ease.OutCubic;
+            case EasingFunction.TfEasecubic:
+                return Ease.InOutCubic;
+            case EasingFunction.TfEaseinquart:
+                return Ease.InQuart;
+            case EasingFunction.TfEaseoutquart:
+                return Ease.OutQuart;
+            case EasingFunction.TfEasequart:
+                return Ease.InOutQuart;
+            case EasingFunction.TfEaseinquint:
+                return Ease.InQuint;
+            case EasingFunction.TfEaseoutquint:
+                return Ease.OutQuint;
+            case EasingFunction.TfEasequint:
+                return Ease.InOutQuint;
+            case EasingFunction.TfEaseincirc:
+                return Ease.InCirc;
+            case EasingFunction.TfEaseoutcirc:
+                return Ease.OutCirc;
+            case EasingFunction.TfEasecirc:
+                return Ease.InOutCirc;
+            case EasingFunction.TfEaseinback:
+                return Ease.InBack;
+            case EasingFunction.TfEaseoutback:
+                return Ease.OutBack;
+            case EasingFunction.TfEaseback:
+                return Ease.InOutBack;
+            case EasingFunction.TfLinear:
+            default:
+                return Ease.Linear;
```

Comment:
To avoid this huge switch we could just make a `Dictionary<EasingFunction, Ease>` and use that as a map, then we could replace this switch with a TryGet, WDYT?

Reply:
Done!